components/brave_news/browser/publishers_controller_unittest.cc

```diff
@@ -341,10 +340,12 @@ TEST_F(BraveNewsPublishersControllerTest, NoPreferredLocale_ReturnsFirstMatch) {
   EXPECT_EQ("en_US", locale);
 
   auto publisher = GetPublisherForSite(GURL("https://tp1.example.com/"));
-  EXPECT_EQ("111", publisher->publisher_id);
+  EXPECT_TRUE(publisher->publisher_id == "111" ||
+              publisher->publisher_id == "222");
 
   publisher = GetPublisherForFeed(GURL("https://tp1.example.com/feed"));
-  EXPECT_EQ("111", publisher->publisher_id);
+  EXPECT_TRUE(publisher->publisher_id == "111" ||
+              publisher->publisher_id == "222");
 }
```

Comment:
This reads a bit weird now - maybe we should just check its not the empty string or something like that?